### PR TITLE
ActiveRecord::SchemaDumper.ignore_tables work on generating dumps

### PR DESCRIPTION
**Pull request by Saicheg**
Hello!

There is an option on AR called ignore_table exists: http://apidock.com/rails/ActiveRecord/SchemaDumper/ignore_tables/class

Right now after setting this option tables you want to remove from your schema.rb dissapear, but if any of  those tables have triggers - they stay there.

This PR make SchemaDumper to take in consideration this option.

### CODE REVIEW

File: lib/hair_trigger/schema_dumper.rb [hunk 1]
```diff
@@ -13,9 +13,11 @@ def triggers(stream)
       @connection = ActiveRecord::Base.connection
       @adapter_name = @connection.adapter_name.downcase.to_sym
 
-      db_triggers = @connection.triggers; nil
```
Comment (Saicheg) [inline lib/hair_trigger/schema_dumper.rb]: @jenseng also i did not understand why we have `;nil` here

**Comment by Saicheg** [comment]
@jenseng also please take a look on this commit

https://github.com/Saicheg/hair_trigger/commit/d26dbc46c80be00f3c6c83fb79faec36b3600463

File: lib/hair_trigger/schema_dumper.rb (hunk 1 above)
Comment (jenseng) [inline lib/hair_trigger/schema_dumper.rb]: i don't either, my guess it's probably an irb debugging artifact that made its way in when i wrote that commit

**Comment by jenseng** [comment]
looks good at first glance. i'll check it out more tonight and hopefully merge. thanks!

**Comment by Saicheg** [comment]
@jenseng actually i have a little problem here, can we talk in skype or somewhere - i have an idea, but just want to clarify with you!

**Comment by jenseng** [comment]
@Saicheg we could talk via google+/hangouts/chat ... i'm jenseng [ at ] gmail.com

**Comment by Saicheg** [comment]
@jenseng any updates on this PR?

**Comment by jenseng** [comment]
Oops, sorry about that. I thought you wanted to clarify something on skype first.

At any rate, this looks fine. I'll merge this and ship a new version. Let me know if you want to want to sync up about this on gchat :+1:

**Comment by jenseng** [comment]
v0.2.14 has been released. thanks again!

**Pull request merged**